packages/wonder-blocks-data/src/hooks/use-data.js

```diff
@@ -73,11 +74,30 @@ export const useDataInternal = <TOptions, TData: ValidData>(
             setResult(null);
         }
 
+        const getMaybeInterceptedHandler = () => {
+            if (interceptor == null) {
+                return handler;
+            }
+
+            const fulfillRequestFn = (options) =>
+                interceptor.fulfillRequest(options) ??
+                handler.fulfillRequest(options);
+            return {
+                fulfillRequest: fulfillRequestFn,
+                getKey: (options) => handler.getKey(options),
```

Comment:
couldn't this be `getKey: handler.getKey`?

Reply:
Flow will complain about detaching the method from the object if we do that.